### PR TITLE
05/viewandindividualreview

### PR DESCRIPTION
**Pull request by code-emad**
task 05 
single review search found on /review
id can be input in to box and submitted and will show review on page
this will also change the url to /review/:review_id

no CSS done

### CODE REVIEW

**Review by xViolaine — CHANGES_REQUESTED** [review]
Ideally, we want our All Reviews not to show the complete article. Currently, it works fine - but what if we have 1000+ word reviews?

Also, we want our single articles to be accessible by a link from the all reviews-list and we also want to be able to go back to the all reviews-list from a single article. While I think the ID search bar is very creative, I don't think it's very intuitive - ideally, we want our users to have a great user experience without relying on them to go to a specific url that they have to manually type in!

File: src/Utils/api.jsx [hunk 1]
```diff
@@ -7,3 +7,11 @@ export const getReviews = () => {
...
     });
 };
+
+export const getReviewById = (review_id) => {
+  return axios
+    .get(`https://emads-be-project.onrender.com/api/reviews/${review_id}`)
+    .then(({ data }) => {
+      return data;
+    });
+};
```
Comment (Hard-Luck) [inline src/Utils/api.jsx]: These files don't return components so should just be .js

File: src/Components/ReviewCards.jsx [hunk 2]
```diff
@@ -1,4 +1,12 @@
+import { useNavigate } from "react-router-dom";
+
+
 export default function ReviewCards({ reviews }) {
+  const navigate = useNavigate()
+  const handleOnClick = (id) => {
```
Comment (Hard-Luck) [inline src/Components/ReviewCards.jsx:6]: The only thing negative about this box is it doesn't enhance the user experience, the user is very unlikely to know the id number and if they did they could just add it to the URL

**Review by Hard-Luck — APPROVED** [review]
Just change those 2 comments I made then you can merge

**Pull request merged**